spec/unit/shoulda/matchers/active_model/validate_length_of_matcher_spec.rb

```diff
@@ -3,23 +3,31 @@
 describe Shoulda::Matchers::ActiveModel::ValidateLengthOfMatcher, type: :model do
   context 'an attribute with a non-zero minimum length validation' do
     it 'accepts ensuring the correct minimum length' do
-      expect(validating_length(minimum: 4)).
-        to validate_length_of(:attr).is_at_least(4)
+      attribute_types(minimum: 4).each do |options|
```

Comment:
Just a note here - instead of embedding loops inside of tests, we typically place the loops outside of the tests. See the tests for the numericality matcher for how to achieve this. (We can do this when we merge this in as it's a bit complicated.)